README.md

```diff
@@ -74,22 +50,18 @@ success.
 For more information look here:
 [Strategic Initiatives](https://github.com/planetarysoftware/TSC/blob/master/Strategic-Initiatives.md)
 
-## Top-Level WGs and TLPs
-
-* [Working Groups](Working-Groups.md)
-* Top-Level Projects
 
 ## Policy Change Proposal Process
 
 The Planetary Software TSC is chartered to oversee the technical governance of all Top
 Level Projects and Working Groups under the Planetary Software Organization. The TSC
-establishes the default governance, conduct, and licensing policies for all Top
-Level Projects. Top Level Projects and Working Groups have broad powers of
+establishes the default governance, conduct, and licensing policies for all Top-Level 
+Projects. Top-Level Projects and Working Groups have broad powers of
 self-governance.
 
 To propose a change or addition to policies or processes that are intended to
-cover all Top Level Projects and Working Groups in the foundation, a PR should
-be opened in the `planetarysoftware/TSC` repository.
+cover all Top-Level Projects and Working Groups in the Planetary Software Organization,
+a PR should be opened in the `planetarysoftware/TSC` repository.
 
 The pull request can be labeled `tsc-agenda` to request that it be put on the
 agenda for the next TSC meeting.
```

Comment:
Does this PR qualify for this? I thought we were allowing for changes to be made outside of a meeting as long as everyone approves them.

Reply:
Are you saying that you read this sentence as 'requiring' that 'poilcy-change' PRs must be put on the agenda (of which this PR qualifies)?

I read the 'can' in that sentence as being optional.  It could be put on the agenda or not.  So I think I agree that we can do this outside of a meeting.

Our TSC PRs have rules for how they get merged: Regular PRs only need 2 reviews.  This PR, since it changes the Charter, requires a 'majority vote.'  That could certainly be done via voice at a meeting, but I think that for even greater transparency, this could also be done by simply requiring a majority of the TSC to 'approve' the PR (which is what I suggested).  These things are equivalent in my mind.  Do you think they should be different?

However, you're saying 'everyone approves' which is an even higher bar.

Or maybe the simple thing is to just remove this sentence about putting something on the agenda?

Reply:
First, I'm suggesting we remove this line as it's the old policy and I think we're moving away from all changes having to be discussed in a TSC meeting to discussion in PRs like this.

Second, I would recommend everyone approves because we should be aiming for everyone to approve things. We do not have to have 100% approval, but the first goal should be 100% approval and then if that doesn't happen after some work we fall back to a majority.

Reply:
That sounds good to me.  I will push a fix shortly which removes that sentence from the PR.

I like your second point, but would you mind submitting that as a PR after this PR goes through?  We need to discuss how that mechanism might work, timing, etc.  Which I think deserves a second discussion thread, or maybe an Issue first to work out the kinks and then a PR?

Alternately we can work it here.  If so, provide some suggested text for me to incorporate.